host/xtest/pkcs11_1000.c

```diff
@@ -931,21 +931,21 @@ ADBG_CASE_DEFINE(pkcs11, 1003, xtest_pkcs11_test_1003,
 		 "PKCS11: Login to PKCS#11 token");
 
 static CK_ATTRIBUTE cktest_token_object[] = {
-	{ CKA_DECRYPT,	&(CK_BBOOL){CK_TRUE}, sizeof(CK_BBOOL) },
-	{ CKA_TOKEN,	&(CK_BBOOL){CK_TRUE}, sizeof(CK_BBOOL) },
-	{ CKA_MODIFIABLE, &(CK_BBOOL){CK_TRUE}, sizeof(CK_BBOOL) },
-	{ CKA_CLASS,	&(CK_OBJECT_CLASS){CKO_SECRET_KEY},
+	{ CKA_DECRYPT,	&(CK_BBOOL) {CK_TRUE}, sizeof(CK_BBOOL) },
```

Comment:
Sorry, that the wrong way.
We expect no space between cast and casted reference
We also expect space chars after `{` and before `}`.
Here:
```c
	{ CKA_DECRYPT,	&(CK_BBOOL){ CK_TRUE }, sizeof(CK_BBOOL) },
	{ CKA_TOKEN,	&(CK_BBOOL){ CK_TRUE }, sizeof(CK_BBOOL) },
	{ CKA_MODIFIABLE, &(CK_BBOOL){ CK_TRUE }, sizeof(CK_BBOOL) },
	{ CKA_CLASS,	&(CK_OBJECT_CLASS){ CKO_SECRET_KEY },
						sizeof(CK_OBJECT_CLASS) },
	{ CKA_KEY_TYPE,	&(CK_KEY_TYPE){ CKK_AES }, sizeof(CK_KEY_TYPE) },
	{ CKA_VALUE,	(void *)cktest_aes128_key, sizeof(cktest_aes128_key) },
```

Must fix all occurrences :(

Reply:
I tried this. Unfortunately checkpatch still complains

ERROR: space required before the open brace '{'
#53: FILE: host/xtest/pkcs11_1000.c:5484:
+		{ CKA_WRAP, &(CK_BBOOL){ CK_TRUE }, sizeof(CK_BBOOL) },

Reply:
I have added an additional commit adding a space after { and before } in test 1020 for now. It still gives checkpatch error as I stated above but I guess we might need to ignore that. If this looks ok to @etienne-lms and @vesajaaskelainen , then I will squash this change with original test commit.